motioneye/static/img/camera-top-buttons.svg

```diff
@@ -171,23 +171,17 @@
        id="path4153"
        inkscape:connector-curvature="0"
        sodipodi:nodetypes="cc" />
-    <path
-       style="fill:#3498db;fill-opacity:1;stroke:#3498db;stroke-width:2;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
-       d="m 212,1026.3622 10,6 -10,6 z"
-       id="path3819-3"
-       inkscape:connector-curvature="0"
-       sodipodi:nodetypes="cccc" />
```

Comment:
You removed the time-lapse button 😉. Our of curiosity, since I guess you used inkscape to edit this SVG, are there any issues to do so with a minified SVG or does this cause havoc in inkscape? https://github.com/motioneye-project/motioneye/blob/imgbot/motioneye/static/img/camera-top-buttons.svg

Otherwise we might need to keep the original versions available separately for editing, minifying them to frontend assets in a second step.

Reply:
Oops, need to fix that. :flushed:  I guess time-lapse was added after I started my branch...

Reply:
Fixed. I didn't notice that Inkscape would've had any issues with the minification. AFAIK all the tags with namespace of `inkscape` or `sodipodi` are non-essential as long as only the actual image data is relevant (that is, no-one wants to store their preferred workspace settings and stuff in the file).

Reply:
> Fixed. I didn't notice that Inkscape would've had any issues with the minification.

Great, as I thought/hoped.